src/sentry/issues/json_schemas.py

```diff
@@ -154,3 +159,11 @@
     ],
     "additionalProperties": False,
 }
+
+try:
+    EVENT_PAYLOAD_SCHEMA = sentry_kafka_schemas._get_schema("generic-events")["schema"]
+except SchemaNotFound:
+    logger.exception(
+        "Failed to load Events schema from Kafka schemas, using legacy hardcoded schema"
+    )
+    raise
```

Comment:
Why do we raise here? This will probably stop the app from loading if it happens. Not sure if it makes sense to either set this to the legacy schema, or to `None` and have a check later on?

Reply:
changed